guides/common/attributes-base.adoc

```diff
@@ -191,10 +192,14 @@
 :client-salt-minion-repository-url: https://packages.broadcom.com/artifactory/saltproject-rpm/
 // Foreman Server and Smart Proxy Server
 :project-minimum-memory: 4 GB
+// Satellite uses satellite-maintain to wrap all package manager actions
+// but you need to install satellite-maintain (and other tooling) before it's
+// available. There are also other cases where it's not available (yet).
```

Comment:
I suggest to move this comment to attributes-satellite.adoc.

We have three types of attributes: commands on Foreman: "project-...". commands on clients: "client-...", and on the Foreman/Katello platform: "package-..."; see line 203. The third category is necessary for cases when you a) do not have satellite-installer available yet, or for hosts that must on on the same platform as Satellite, e.g. external databases.

Reply:
OK, then let's keep it here.